http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/DbReference.scala

```diff
@@ -105,7 +109,8 @@ final class DataAccess(blocker: Blocker) {
       TableQuery[ClusterImageTable].delete andThen
       TableQuery[ScopeTable].delete andThen
       TableQuery[FollowupTable].delete andThen
-      TableQuery[ClusterTable].delete
+      TableQuery[ClusterTable].delete andThen
+      Queries.runtimeConfigs.delete
```

Comment:
Since `CLUSTER` has a foreign key to `RUNTIME_CONFIG`, should the `RUNTIME_CONFIG` deletion happen first?

Reply:
Wait nevermind… this is correct